### PR TITLE
WebSocketUtils: Generate Sec-WebSocket-Key

### PR DESCRIPTION
**Pull request by ManosMagnus**
Generates 'Sec-WebSocket-Key' which is used in websocket Handshake. 

Open this PR in order to break up [PR#1618](https://github.com/zaproxy/zap-extensions/pull/1618)

### CODE REVIEW

File: src/org/zaproxy/zap/extension/websocket/utility/WebSocketUtils.java [hunk 1]
```diff
@@ -40,5 +42,31 @@ public static String encodeWebSocketKey(String key) {
...
+        byte[] random = new byte[16];
+        Random rand = new Random();
+        rand.nextBytes(random);
+        return Base64.getEncoder().encodeToString(random);
+    }
+    
+    /**
+     * Given a web socket key, Generate our accept key
+     */
+    public static String getAcceptKey(String strWebSocketKey) throws NoSuchAlgorithmException {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/websocket/utility/WebSocketUtils.java]: We are already doing this in `encodeWebSocketKey`, worth adding the JavaDoc to that one.

Comment (ManosMagnus) [reply src/org/zaproxy/zap/extension/websocket/utility/WebSocketUtils.java]: I didn't notice that, sorry

Comment (thc202) [reply src/org/zaproxy/zap/extension/websocket/utility/WebSocketUtils.java]: No worries, I didn't either when in the other PR. The commit message (and PR title) should be tweaked though.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**